src/todo/UI.lua

```diff
@@ -324,6 +324,7 @@ function todo.create_add_edit_frame(player, task)
     if task and task.assignee then
         assign_index = lookup[task.assignee]
     elseif todo.is_auto_assign(player) and c == 1 then
+        -- TODO: this should be something like assign_index = lookup[player.name] no?
```

Comment:
No this is actually correct as is. If you are in singleplacer (c == 1) then the list contains always two elements with the player being the second and the first being "unassigned".
Your suggestion would make it more secure, though.

Reply:
Ho that's why my player_index is always 2 in single player.